moz_telemetry/tests/integration/new_experiment_monitor/run/analysis/new_experiment_monitor.cfg

```diff
@@ -0,0 +1,16 @@
+filename = 'moz_telemetry_new_experiment_monitor.lua'
+ticker_interval = 60
+message_matcher = 'Type == "telemetry" && Fields[docType] == "main" && Fields[environment.experiments] != NIL'
+preserve_data = true
+alert = {
+  ['modules'] = {
```

Comment:
The brackets and single quotes aren't needed on any of these variables (you can get rid of the subscript on recipients also)